src/main.rs

```diff
@@ -54,17 +54,31 @@ use transform::*;
 async fn start() {
     let event_loop = winit::event_loop::EventLoop::new();
 
-    let title = format!("{} v{}", env!("CARGO_PKG_NAME"), env!("CARGO_PKG_VERSION"));
-    let window = winit::window::WindowBuilder::new()
-        // .with_inner_size(winit::dpi::LogicalSize::new(1000.0f32, 1000.0f32))
-        // .with_fullscreen(Some(winit::window::Fullscreen::Borderless(None)))
-        .with_inner_size(winit::dpi::PhysicalSize::new(1920.0, 1080.0))
-        .with_title(title)
-        // .with_visible(false)
-        .build(&event_loop)
-        .expect("Failed to create window");
+    let window = {
+        let (width, height) = match event_loop.primary_monitor() {
+            None => (1920, 1080), // Most widespread resolution in 2022.
+            Some(handle) => (handle.size().width, handle.size().height),
+        };
+        let inner_size = winit::dpi::LogicalSize::new(width /2, height /2);
```

Comment:
on my computer the logical size causes the window to be quite small. To reproduce the old behavior I had to change this to:

```suggestion
        let inner_size = winit::dpi::PhysicalSize::new(width * 3 / 4, height * 3 / 4);
```

Does that work well on your computer?

Reply:
Works without problems!